ciao/cmd/add.go

```diff
@@ -0,0 +1,902 @@
+// Copyright © 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/ciao-project/ciao/ciao-controller/api"
+	"github.com/ciao-project/ciao/ciao-controller/types"
+	"github.com/ciao-project/ciao/ciao/tool"
+	"github.com/ciao-project/ciao/payloads"
+	"github.com/ciao-project/ciao/uuid"
+
+	"github.com/intel/tfortools"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var config string
+
+var addCmd = &cobra.Command{
+	Use:   "add",
```

Comment:
One thing that we talked about is that "add" has always bugged us. It should probably be create. @kaccardi does that align with your recollection?

Reply:
Up until now I thought everyone was in agreement over which commands are to be merged and which ones will end up being used to perform actions going forward. This seems off track to split things up again, and a large change that should have a written spec if it's going to be done.